test/isIsogram.test.js

```diff
@@ -0,0 +1,46 @@
+import { equal } from 'assert';
+import { isIsogram } from '../isIsogram.js';
+
+it('Is an Isogram - Dermatoglyphics', () => {
```

Comment:
Consider adding describes to group similar test cases together

Reply:
- Is an Isogram
  - Dermatoglyphics
  - ant
- Not an Isogram
  - aba
  - Different casing Aba
  - etc.
- other tests

Reply:
Adding describes makes test runs a lot clearer :)
![image](https://github.com/kirithecat/MoniquesPlaygroundOne/assets/36695580/d0ed5eff-3b7e-481f-a964-d3c368ff22be)